plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -521,33 +532,40 @@ class DefaultPsiToDocumentableTranslator(
                                 type.parameters.map { getProjection(it) },
                                 extra = type.annotations()
                             )
-                            else -> GenericTypeConstructor(
-                                DRI.from(resolved),
-                                type.parameters.map { getProjection(it) },
-                                extra = type.annotations()
-                            )
+
+                            else -> {
+                                // cache types that has no annotation and no type parameter
```

Comment:
Side note: this comments explains the code, but doesn't explain the intention behind it, so it's not really clear why we can't cache type parameters. Describing a use case where this can hurt would be nice